package.json

```diff
@@ -48,7 +48,7 @@
     "gulp-tslint": "^5.0.0",
     "gulp-util": "^3.0.6",
     "gulp-watch": "^4.3.4",
-    "harp": "git://github.com/filipesilva/harp.git#8da8d3497ddbfcbcbadd8be63e0fd731d7310cc4",
+    "harp": "^0.21.0-pre.0",
```

Comment:
I believe it's pre.1 instead, see https://github.com/filipesilva/harp/commit/bf9030cb1ac2a6035bae3da9afbdab4990011370#commitcomment-19049717

I would also advise to not use the caret, semver resolution falls back to alphabetical on these non-semver ones.